### ISSUE TITLE
BigQuery not working in 0.36.8/0.37.0

### ISSUE THREAD

**Issue by marcoruggine** [metabase/metabase#13582]
**Describe the bug**
I simply updated to 0.37.0 via docker, went back to my queries and suddenly I had this error

<img width="549" alt="Screenshot 2020-10-23 at 17 24 47" src="https://user-images.githubusercontent.com/20041428/97025570-8a81cd80-1558-11eb-8150-da28ba0fc1a1.png">

So I went back, reverted to 0.36.8 and the error still remains.

I've checked under my databases and it appears that Google JSON is missing, so I've tried several time to re-upload but, even if the button gets green and says it did save the file, when reloading the page my JSON is still missing

**Logs**
```
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 INFO metabase.driver.impl Registered abstract driver :google 🚚
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 DEBUG metabase.plugins.lazy-loaded-driver Registering lazy loading driver :google...
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 INFO metabase.plugins.dependencies Metabase BigQuery Driver dependency {:plugin Metabase Google Drivers Shared Dependencies} satisfied? true
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 INFO metabase.plugins.dependencies Metabase Google Analytics Driver dependency {:plugin Metabase Google Drivers Shared Dependencies} satisfied? true
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 DEBUG metabase.plugins.initialize Dependencies satisfied; these plugins will now be loaded: ["Metabase Google Analytics Driver" "Metabase BigQuery Driver"]
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 INFO metabase.driver.impl Registered driver :googleanalytics (parents: [:google]) 🚚
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 DEBUG metabase.plugins.lazy-loaded-driver Registering lazy loading driver :googleanalytics...
[6b8c138a-ff1b-4bb0-acb1-432bf18d616e] 2020-10-23T18:09:33+02:00 WARN metabase.plugins.lazy-loaded-driver Warning: plugin manifest for :bigquery does not include connection properties

[...]

at java.base/java.lang.Thread.run(Unknown Source)
[2f57ae5e-a983-4726-90e4-7e38d661f06e] 2020-10-23T17:58:19+02:00 ERROR metabase.models.field-values Error fetching field values
clojure.lang.ExceptionInfo: Not found: Job playtomic-bi:job_7kq482M6K4BPiVTnbYDctFydQZ5G {"code" 404, "errors" [{"domain" "global", "message" "Not found: Job playtomic-bi:job_7kq482M6K4BPiVTnbYDctFydQZ5G", "reason" "notFound"}], "message" "Not found: Job playtomic-bi:job_7kq482M6K4BPiVTnbYDctFydQZ5G", "status" "NOT_FOUND"}
	at metabase.driver.google$execute_no_auto_retry.invokeStatic(google.clj:42)
	at metabase.driver.google$execute_no_auto_retry.invoke(google.clj:34)
	at metabase.driver.google$execute.invokeStatic(google.clj:56)
	at metabase.driver.google$execute.invoke(google.clj:46)
	at metabase.driver.bigquery$get_query_results.invokeStatic(bigquery.clj:193)
	at metabase.driver.bigquery$get_query_results.invoke(bigquery.clj:186)
	at metabase.driver.bigquery$execute_bigquery.invokeStatic(bigquery.clj:211)
	at metabase.driver.bigquery$execute_bigquery.invoke(bigquery.clj:195)
	at metabase.driver.bigquery$execute_bigquery.invokeStatic(bigquery.clj:197)
	at metabase.driver.bigquery$execute_bigquery.invoke(bigquery.clj:195)
	at metabase.driver.bigquery$process_native_STAR_$thunk__1541.invoke(bigquery.clj:261)
	at metabase.driver.bigquery$process_native_STAR_.invokeStatic(bigquery.clj:266)
	at metabase.driver.bigquery$process_native_STAR_.invoke(bigquery.clj:256)
	at metabase.driver.bigquery$eval1545$fn__1547.invoke(bigquery.clj:280)
	at clojure.lang.MultiFn.invoke(MultiFn.java:244)
	at metabase.query_processor.context$executef.invokeStatic(context.clj:59)
	at metabase.query_processor.context$executef.invoke(context.clj:48)
	at metabase.query_processor.context.default$default_runf.invokeStatic(default.clj:69)
	at metabase.query_processor.context.default$default_runf.invoke(default.clj:67)
	at metabase.query_processor.context$runf.invokeStatic(context.clj:45)
	at metabase.query_processor.context$runf.invoke(context.clj:39)
	at metabase.query_processor.reducible$pivot.invokeStatic(reducible.clj:34)
	at metabase.query_processor.reducible$pivot.invoke(reducible.clj:31)
	at metabase.query_processor.middleware.mbql_to_native$mbql__GT_native$fn__45880.invoke(mbql_to_native.clj:26)
	at metabase.query_processor.middleware.check_features$check_features$fn__45155.invoke(check_features.clj:42)
	at metabase.query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__46045.invoke(optimize_datetime_filters.clj:133)
	at metabase.query_processor.middleware.auto_parse_filter_values$auto_parse_filter_values$fn__43960.invoke(auto_parse_filter_values.clj:44)
	at metabase.query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__47597.invoke(wrap_value_literals.clj:137)
	at metabase.query_processor.middleware.annotate$add_column_info$fn__43722.invoke(annotate.clj:574)
	at metabase.query_processor.middleware.permissions$check_query_permissions$fn__45030.invoke(permissions.clj:64)
	at metabase.query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__46563.invoke(pre_alias_aggregations.clj:40)
	at metabase.query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__45228.invoke(cumulative_aggregations.clj:61)
	at metabase.query_processor.middleware.resolve_joined_fields$resolve_joined_fields$fn__46800.invoke(resolve_joined_fields.clj:36)
	at metabase.query_processor.middleware.resolve_joins$resolve_joins$fn__47119.invoke(resolve_joins.clj:183)
	at metabase.query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__39448.invoke(add_implicit_joins.clj:254)
	at metabase.query_processor.middleware.large_int_id$convert_id_to_string$fn__45841.invoke(large_int_id.clj:44)
	at metabase.query_processor.middleware.limit$limit$fn__45866.invoke(limit.clj:38)
	at metabase.query_processor.middleware.format_rows$format_rows$fn__45821.invoke(format_rows.clj:84)
	at metabase.query_processor.middleware.desugar$desugar$fn__45294.invoke(desugar.clj:22)
	at metabase.query_processor.middleware.binning$update_binning_strategy$fn__44319.invoke(binning.clj:229)
	at metabase.query_processor.middleware.resolve_fields$resolve_fields$fn__44836.invoke(resolve_fields.clj:24)
	at metabase.query_processor.middleware.add_dimension_projections$add_remapping$fn__38992.invoke(add_dimension_projections.clj:318)
	at metabase.query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__39199.invoke(add_implicit_clauses.clj:141)
	at metabase.query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__39597.invoke(add_source_metadata.clj:105)
	at metabase.query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__46760.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)
	at metabase.query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__43907.invoke(auto_bucket_datetimes.clj:125)
	at metabase.query_processor.middleware.resolve_source_table$resolve_source_tables$fn__44883.invoke(resolve_source_table.clj:46)
	at metabase.query_processor.middleware.parameters$substitute_parameters$fn__46545.invoke(parameters.clj:114)
	at metabase.query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__44935.invoke(resolve_referenced.clj:80)
	at metabase.query_processor.middleware.expand_macros$expand_macros$fn__45550.invoke(expand_macros.clj:158)
	at metabase.query_processor.middleware.add_timezone_info$add_timezone_info$fn__39628.invoke(add_timezone_info.clj:15)
	at metabase.query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__47481.invoke(splice_params_in_response.clj:32)
	at metabase.query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46771$fn__46775.invoke(resolve_database_and_driver.clj:33)
	at metabase.driver$do_with_driver.invokeStatic(driver.clj:61)
	at metabase.driver$do_with_driver.invoke(driver.clj:57)
	at metabase.query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46771.invoke(resolve_database_and_driver.clj:27)
	at metabase.query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__45768.invoke(fetch_source_query.clj:267)
	at metabase.query_processor.middleware.store$initialize_store$fn__47490$fn__47491.invoke(store.clj:11)
	at metabase.query_processor.store$do_with_store.invokeStatic(store.clj:46)
	at metabase.query_processor.store$do_with_store.invoke(store.clj:40)
	at metabase.query_processor.middleware.store$initialize_store$fn__47490.invoke(store.clj:10)
	at metabase.query_processor.middleware.cache$maybe_return_cached_results$fn__44812.invoke(cache.clj:214)
	at metabase.query_processor.middleware.validate$validate_query$fn__47499.invoke(validate.clj:10)
	at metabase.query_processor.middleware.normalize_query$normalize$fn__45893.invoke(normalize_query.clj:22)
	at metabase.query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__39466.invoke(add_rows_truncated.clj:36)
	at metabase.query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__47466.invoke(results_metadata.clj:147)
	at metabase.query_processor.reducible$async_qp$qp_STAR___38255$thunk__38256.invoke(reducible.clj:101)
	at metabase.query_processor.reducible$async_qp$qp_STAR___38255.invoke(reducible.clj:107)
	at metabase.query_processor.reducible$async_qp$qp_STAR___38255.invoke(reducible.clj:94)
	at metabase.query_processor.reducible$sync_qp$qp_STAR___38264$fn__38265.invoke(reducible.clj:129)
	at metabase.query_processor.reducible$sync_qp$qp_STAR___38264.invoke(reducible.clj:128)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.query_processor$process_query.invokeStatic(query_processor.clj:128)
	at metabase.query_processor$process_query.doInvoke(query_processor.clj:122)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at metabase.db.metadata_queries$qp_query$fn__52646.invoke(metadata_queries.clj:19)
	at metabase.db.metadata_queries$qp_query.invokeStatic(metadata_queries.clj:18)
	at metabase.db.metadata_queries$qp_query.invoke(metadata_queries.clj:16)
	at metabase.db.metadata_queries$field_query.invokeStatic(metadata_queries.clj:28)
	at metabase.db.metadata_queries$field_query.invoke(metadata_queries.clj:26)
	at metabase.db.metadata_queries$fn__52666$field_distinct_values__52675$fn__52678.invoke(metadata_queries.clj:73)
	at metabase.db.metadata_queries$fn__52666$field_distinct_values__52675.invoke(metadata_queries.clj:66)
	at metabase.db.metadata_queries$fn__52666$field_distinct_values__52675$fn__52676.invoke(metadata_queries.clj:70)
	at metabase.db.metadata_queries$fn__52666$field_distinct_values__52675.invoke(metadata_queries.clj:66)
	at clojure.lang.Var.invoke(Var.java:384)
	at metabase.models.field_values$distinct_values.invokeStatic(field_values.clj:79)
	at metabase.models.field_values$distinct_values.invoke(field_values.clj:73)
	at metabase.models.field_values$create_or_update_field_values_BANG_.invokeStatic(field_values.clj:101)
	at metabase.models.field_values$create_or_update_field_values_BANG_.doInvoke(field_values.clj:95)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at metabase.sync.field_values$fn__53239$update_field_values_for_field_BANG___53244$fn__53245.invoke(field_values.clj:25)
	at metabase.sync.field_values$fn__53239$update_field_values_for_field_BANG___53244.invoke(field_values.clj:23)
	at metabase.sync.field_values$fn__53265$update_field_values_for_table_BANG___53270$fn__53271$fn__53272$fn__53273.invoke(field_values.clj:46)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:149)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:142)
	at metabase.sync.field_values$fn__53265$update_field_values_for_table_BANG___53270$fn__53271$fn__53272.invoke(field_values.clj:44)
	at clojure.lang.PersistentVector.reduce(PersistentVector.java:343)
	at clojure.core$reduce.invokeStatic(core.clj:6827)
	at clojure.core$reduce.invoke(core.clj:6810)
	at metabase.sync.field_values$fn__53265$update_field_values_for_table_BANG___53270$fn__53271.invoke(field_values.clj:43)
	at metabase.sync.field_values$fn__53265$update_field_values_for_table_BANG___53270.invoke(field_values.clj:40)
	at clojure.core$map$fn__5866.invoke(core.clj:2753)
	at clojure.lang.LazySeq.sval(LazySeq.java:42)
	at clojure.lang.LazySeq.seq(LazySeq.java:51)
	at clojure.lang.ChunkedCons.chunkedNext(ChunkedCons.java:59)
	at clojure.core$chunk_next.invokeStatic(core.clj:708)
	at clojure.core$reduce1.invokeStatic(core.clj:942)
	at clojure.core$reduce1.invokeStatic(core.clj:934)
	at clojure.core$merge_with.invokeStatic(core.clj:3059)
	at clojure.core$merge_with.doInvoke(core.clj:3051)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.sync.field_values$fn__53292$update_field_values_for_database_BANG___53297$fn__53298.invoke(field_values.clj:54)
	at metabase.sync.field_values$fn__53292$update_field_values_for_database_BANG___53297.invoke(field_values.clj:52)
	at metabase.sync.util$fn__42481$run_step_with_metadata__42486$fn__42490$fn__42492.invoke(util.clj:345)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:102)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:96)
	at metabase.sync.util$with_start_and_finish_debug_logging.invokeStatic(util.clj:119)
	at metabase.sync.util$with_start_and_finish_debug_logging.invoke(util.clj:116)
	at metabase.sync.util$fn__42481$run_step_with_metadata__42486$fn__42490.invoke(util.clj:342)
	at metabase.sync.util$fn__42481$run_step_with_metadata__42486.invoke(util.clj:337)
	at metabase.sync.util$fn__42673$run_sync_operation__42678$fn__42679$fn__42680.invoke(util.clj:430)
	at clojure.core$mapv$fn__8445.invoke(core.clj:6912)
	at clojure.lang.PersistentVector.reduce(PersistentVector.java:343)
	at clojure.core$reduce.invokeStatic(core.clj:6827)
	at clojure.core$mapv.invokeStatic(core.clj:6903)
	at clojure.core$mapv.invoke(core.clj:6903)
	at metabase.sync.util$fn__42673$run_sync_operation__42678$fn__42679.invoke(util.clj:430)
	at metabase.sync.util$fn__42673$run_sync_operation__42678.invoke(util.clj:424)
	at metabase.sync.field_values$fn__53318$update_field_values_BANG___53323$fn__53324$fn__53325.invoke(field_values.clj:69)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:149)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:142)
	at metabase.sync.util$do_with_error_handling.invokeStatic(util.clj:145)
	at metabase.sync.util$do_with_error_handling.invoke(util.clj:142)
	at clojure.core$partial$fn__5839.invoke(core.clj:2623)
	at metabase.driver$fn__22480.invokeStatic(driver.clj:540)
	at metabase.driver$fn__22480.invoke(driver.clj:540)
	at clojure.lang.MultiFn.invoke(MultiFn.java:239)
	at metabase.sync.util$sync_in_context$fn__42393.invoke(util.clj:138)
	at metabase.sync.util$with_db_logging_disabled$fn__42390.invoke(util.clj:129)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invokeStatic(util.clj:102)
	at metabase.sync.util$with_start_and_finish_logging_STAR_.invoke(util.clj:96)
	at metabase.sync.util$with_start_and_finish_logging$fn__42379.invoke(util.clj:114)
	at metabase.sync.util$with_sync_events$fn__42374.invoke(util.clj:88)
	at metabase.sync.util$with_duplicate_ops_prevented$fn__42365.invoke(util.clj:67)
	at metabase.sync.util$do_sync_operation.invokeStatic(util.clj:164)
	at metabase.sync.util$do_sync_operation.invoke(util.clj:161)
	at metabase.sync.field_values$fn__53318$update_field_values_BANG___53323$fn__53324.invoke(field_values.clj:67)
	at metabase.sync.field_values$fn__53318$update_field_values_BANG___53323.invoke(field_values.clj:63)
	at metabase.api.database$fn__55638$fn__55640.invoke(database.clj:636)
	at clojure.core$binding_conveyor_fn$fn__5754.invoke(core.clj:2030)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

**Severity**
🔥🔥🔥 Extremely severe 🔥🔥🔥 I'd say Bruce Willis in Armageddon severity!
My whole company doesn't have any analytical tool, alerts, operational dashboard to assist our users...

**Metabase Diagnostic Info**

```json
{
  "browser-info": {
    "language": "en",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.80 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.7+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.7",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.7+10",
    "os.name": "Linux",
    "os.version": "4.9.0-12-amd64",
    "user.language": "en",
    "user.timezone": "Europe/Berlin"
  },
  "metabase-info": {
    "databases": [
      "mysql",
      "postgres",
      "bigquery"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "9.6.17"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-10-22",
      "tag": "v0.36.8",
      "branch": "release-0.36.x",
      "hash": "e49cdb1"
    },
    "settings": {
      "report-timezone": "Europe/Berlin"
    }
  }
}
```

**Comment by camsaul** [comment]
Hi @marcoruggine, can you trying using 0.36.7?

**Comment by marcoruggine** [comment]
> Hi @marcoruggine, can you trying using 0.36.7?

I was just getting back here to update: 0.36.7 is running well!

**Comment by flamber** [comment]
Just so the "missing JSON after save" isn't causing a wild goose chase - that's #13442

**Comment by camsaul** [comment]
Hey @marcoruggine, we have a fix for this issue in #13584 -- do you think you'd be able to try it for us and confirm it works? We can build an uberjar or Docker image if that would be more convenient for you

**Comment by camsaul** [comment]
Fixed by #13584

**Comment by tomalexander** [comment]
This is still happening for us on metabase 0.37.0.1

**Comment by camsaul** [comment]
@tomalexander the "Not found" error?

**Comment by tomalexander** [comment]
yep. Version:
```
Thanks for using Metabase!
You're on version v0.37.0.1
Built on 2020-10-23
Branch: release-x.37.x
Hash: 13dd8f9
```

Stacktrace:
```
[5d3c2e19-9386-44cc-bd8a-97aa650efa25] 2020-10-23T17:48:43-04:00 ERROR metabase.query-processor.middleware.catch-exceptions Error processing query: null
{:database_id 5,
 :started_at #t "2020-10-23T21:48:38.116986Z[GMT]",
 :json_query
 {:type "native",
  :native {:query "SELECT count(*) FROM `segment-warehouse-aa060fae.sendgrid.activity`", :template-tags {}},
  :database 5,
  :parameters [],
  :middleware {:js-int-to-string? true, :add-default-userland-constraints? true}},
 :status :failed,
 :class clojure.lang.ExceptionInfo,
 :stacktrace
 ["--> driver.google$execute_no_auto_retry.invokeStatic(google.clj:42)"
  "driver.google$execute_no_auto_retry.invoke(google.clj:34)"
  "driver.google$execute.invokeStatic(google.clj:56)"
  "driver.google$execute.invoke(google.clj:46)"
  "driver.bigquery$get_query_results.invokeStatic(bigquery.clj:204)"
  "driver.bigquery$get_query_results.invoke(bigquery.clj:197)"
  "driver.bigquery$execute_bigquery$fn__1544.invoke(bigquery.clj:223)"
  "driver.bigquery$do_with_finished_response.invokeStatic(bigquery.clj:175)"
  "driver.bigquery$do_with_finished_response.invoke(bigquery.clj:166)"
  "driver.bigquery$execute_bigquery.invokeStatic(bigquery.clj:222)"
  "driver.bigquery$execute_bigquery.invoke(bigquery.clj:206)"
  "driver.bigquery$execute_bigquery.invokeStatic(bigquery.clj:208)"
  "driver.bigquery$execute_bigquery.invoke(bigquery.clj:206)"
  "driver.bigquery$process_native_STAR_$thunk__1645.invoke(bigquery.clj:273)"
  "driver.bigquery$process_native_STAR_.invokeStatic(bigquery.clj:278)"
  "driver.bigquery$process_native_STAR_.invoke(bigquery.clj:268)"
  "driver.bigquery$eval1649$fn__1651.invoke(bigquery.clj:292)"
  "query_processor.context$executef.invokeStatic(context.clj:59)"
  "query_processor.context$executef.invoke(context.clj:48)"
  "query_processor.context.default$default_runf.invokeStatic(default.clj:69)"
  "query_processor.context.default$default_runf.invoke(default.clj:67)"
  "query_processor.context$runf.invokeStatic(context.clj:45)"
  "query_processor.context$runf.invoke(context.clj:39)"
  "query_processor.reducible$pivot.invokeStatic(reducible.clj:34)"
  "query_processor.reducible$pivot.invoke(reducible.clj:31)"
  "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__45926.invoke(mbql_to_native.clj:26)"
  "query_processor.middleware.check_features$check_features$fn__45201.invoke(check_features.clj:42)"
  "query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__46091.invoke(optimize_datetime_filters.clj:133)"
  "query_processor.middleware.auto_parse_filter_values$auto_parse_filter_values$fn__44006.invoke(auto_parse_filter_values.clj:44)"
  "query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__47657.invoke(wrap_value_literals.clj:149)"
  "query_processor.middleware.annotate$add_column_info$fn__43768.invoke(annotate.clj:574)"
  "query_processor.middleware.permissions$check_query_permissions$fn__45076.invoke(permissions.clj:64)"
  "query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__46609.invoke(pre_alias_aggregations.clj:40)"
  "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__45274.invoke(cumulative_aggregations.clj:61)"
  "query_processor.middleware.resolve_joined_fields$resolve_joined_fields$fn__46846.invoke(resolve_joined_fields.clj:36)"
  "query_processor.middleware.resolve_joins$resolve_joins$fn__47165.invoke(resolve_joins.clj:183)"
  "query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__39493.invoke(add_implicit_joins.clj:254)"
  "query_processor.middleware.large_int_id$convert_id_to_string$fn__45887.invoke(large_int_id.clj:44)"
  "query_processor.middleware.limit$limit$fn__45912.invoke(limit.clj:38)"
  "query_processor.middleware.format_rows$format_rows$fn__45867.invoke(format_rows.clj:84)"
  "query_processor.middleware.desugar$desugar$fn__45340.invoke(desugar.clj:22)"
  "query_processor.middleware.binning$update_binning_strategy$fn__44365.invoke(binning.clj:229)"
  "query_processor.middleware.resolve_fields$resolve_fields$fn__44882.invoke(resolve_fields.clj:24)"
  "query_processor.middleware.add_dimension_projections$add_remapping$fn__39037.invoke(add_dimension_projections.clj:316)"
  "query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__39244.invoke(add_implicit_clauses.clj:141)"
  "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__39642.invoke(add_source_metadata.clj:105)"
  "query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__46806.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)"
  "query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__43953.invoke(auto_bucket_datetimes.clj:125)"
  "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__44929.invoke(resolve_source_table.clj:46)"
  "query_processor.middleware.parameters$substitute_parameters$fn__46591.invoke(parameters.clj:114)"
  "query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__44981.invoke(resolve_referenced.clj:80)"
  "query_processor.middleware.expand_macros$expand_macros$fn__45596.invoke(expand_macros.clj:158)"
  "query_processor.middleware.add_timezone_info$add_timezone_info$fn__39673.invoke(add_timezone_info.clj:15)"
  "query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__47527.invoke(splice_params_in_response.clj:32)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46817$fn__46821.invoke(resolve_database_and_driver.clj:33)"
  "driver$do_with_driver.invokeStatic(driver.clj:61)"
  "driver$do_with_driver.invoke(driver.clj:57)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46817.invoke(resolve_database_and_driver.clj:27)"
  "query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__45814.invoke(fetch_source_query.clj:267)"
  "query_processor.middleware.store$initialize_store$fn__47536$fn__47537.invoke(store.clj:11)"
  "query_processor.store$do_with_store.invokeStatic(store.clj:46)"
  "query_processor.store$do_with_store.invoke(store.clj:40)"
  "query_processor.middleware.store$initialize_store$fn__47536.invoke(store.clj:10)"
  "query_processor.middleware.cache$maybe_return_cached_results$fn__44858.invoke(cache.clj:214)"
  "query_processor.middleware.validate$validate_query$fn__47545.invoke(validate.clj:10)"
  "query_processor.middleware.normalize_query$normalize$fn__45939.invoke(normalize_query.clj:22)"
  "query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__39511.invoke(add_rows_truncated.clj:36)"
  "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__47512.invoke(results_metadata.clj:147)"
  "query_processor.middleware.constraints$add_default_userland_constraints$fn__45217.invoke(constraints.clj:42)"
  "query_processor.middleware.process_userland_query$process_userland_query$fn__46680.invoke(process_userland_query.clj:136)"
  "query_processor.middleware.catch_exceptions$catch_exceptions$fn__45160.invoke(catch_exceptions.clj:174)"
  "query_processor.reducible$async_qp$qp_STAR___38300$thunk__38301.invoke(reducible.clj:101)"
  "query_processor.reducible$async_qp$qp_STAR___38300.invoke(reducible.clj:107)"
  "query_processor.reducible$sync_qp$qp_STAR___38309$fn__38312.invoke(reducible.clj:133)"
  "query_processor.reducible$sync_qp$qp_STAR___38309.invoke(reducible.clj:132)"
  "query_processor$process_userland_query.invokeStatic(query_processor.clj:219)"
  "query_processor$process_userland_query.doInvoke(query_processor.clj:215)"
  "query_processor$fn__47701$process_query_and_save_execution_BANG___47710$fn__47713.invoke(query_processor.clj:231)"
  "query_processor$fn__47701$process_query_and_save_execution_BANG___47710.invoke(query_processor.clj:223)"
  "query_processor$fn__47745$process_query_and_save_with_max_results_constraints_BANG___47754$fn__47757.invoke(query_processor.clj:243)"
  "query_processor$fn__47745$process_query_and_save_with_max_results_constraints_BANG___47754.invoke(query_processor.clj:236)"
  "api.dataset$fn__51065$fn__51068.invoke(dataset.clj:55)"
  "query_processor.streaming$streaming_response_STAR_$fn__35712$fn__35713.invoke(streaming.clj:73)"
  "query_processor.streaming$streaming_response_STAR_$fn__35712.invoke(streaming.clj:72)"
  "async.streaming_response$do_f_STAR_.invokeStatic(streaming_response.clj:66)"
  "async.streaming_response$do_f_STAR_.invoke(streaming_response.clj:64)"
  "async.streaming_response$do_f_async$fn__23308.invoke(streaming_response.clj:85)"],
 :context :ad-hoc,
 :error "Not found: Job segment-warehouse-aa060fae:job_7kSxc6s-6NYuPBdfIqQFK2xyTsnk",
 :row_count 0,
 :running_time 0,
 :ex-data
 {"code" 404,
  "errors" [{"domain" "global", "message" "Not found: Job segment-warehouse-aa060fae:job_7kSxc6s-6NYuPBdfIqQFK2xyTsnk", "reason" "notFound"}],
  "message" "Not found: Job segment-warehouse-aa060fae:job_7kSxc6s-6NYuPBdfIqQFK2xyTsnk",
  "status" "NOT_FOUND"},
 :data {:rows [], :cols []}}
```

**Comment by robdaemon** [comment]
@tomalexander Can you give us a little more detail on your setup?

1. Are you using a service account for BigQuery?
1. What region is your data in?

**Comment by tomalexander** [comment]
1. Yep, we're using a service account to authenticate to BigQuery
2. Our BigQuery data is in `us-east1` but our metabase instances are running in GKE in `us-central1`

**Comment by robdaemon** [comment]
@tomalexander thank you! The region was the trick here - it only happens in `useast1`, which meant we could reproduce it to fix it.

**Comment by robdaemon** [comment]
@tomalexander @marcoruggine can you try it again with a build off of this fix? https://github.com/metabase/metabase/pull/13592

We can create a Docker image or jar for you, if necessary.

**Comment by marcoruggine** [comment]
@robdaemon @camsaul sorry for my late reply, it was Friday night here 😄 
If you could create a Docker image it would be great, I'll try my best to find some time today to test it

**Comment by tomalexander** [comment]
Similar situation, sorry for my late reply, it was Friday night here :-D.
Yeah if you could create a docker image that would be the simplest way for me to get that into our GKE cluster for testing. I'll also be able to test it today.

**Comment by flamber** [comment]
@tomalexander @marcoruggine Would you be able to test Monday evening your time (I guess you're both in Europe)?
Both Cam and Rob are in PST timezone, so that would be morning for them.

**Comment by marcoruggine** [comment]
@flamber sure! I'll try Monday late afternoon. I'm in Italy so 18:00 CET should be 09:00 PST, anyways as soon as I get a docker image I can test it.

**Comment by camsaul** [comment]
@marcoruggine We've pushed Docker image `metabase/metabase:v0.36.8.2-rc` with the fix from #13592. Please try it and let us know if it works for you!

**Comment by tomalexander** [comment]
Should I also give that a try on our cluster or would going back to a 0.36* release break after upgrading to 0.37.0.1?

**Comment by robdaemon** [comment]
@tomalexander You can give it a shot if you like - the fix is that we set the region in the pagination requests now. The API docs imply that shouldn't be necessary for US and EU regions, but that seems to exclude `useast1`

Note: this fix is in 0.36.8 and 0.37.0, so if you do decide to revert, you'll want to go to 0.36.7

**Comment by tomalexander** [comment]
The issue is fixed for us on the `metabase/metabase:v0.36.8.2-rc` image, thanks!

**Comment by camsaul** [comment]
Fixed by #13592

**Comment by marcoruggine** [comment]
Guys I'm sorry but my boss 'kidnapped' me for a we_need_it_yesterday_project and couldn't work on this
